zerver/lib/integrations.py

```diff
@@ -148,6 +148,7 @@ def url_object(self):
     WebhookIntegration('pivotal', display_name='Pivotal Tracker'),
     WebhookIntegration('semaphore'),
     WebhookIntegration('sentry'),
+    WebhookIntegration('slack', display_name='Slack'),
```

Comment:
display name where only first letter is capitalize should be done by default, so display_name can be removed I think